Dockerfile

```diff
@@ -1,16 +1,15 @@
-# Docker image for the Drone SFTP plugin
-
 FROM alpine:3.3
 
 RUN apk update && \
-    apk add nodejs && \
-    apk add git && \
```

Comment:
I believe that some dependencies from NPM are pulling things from Git. That's why I had to put it there